Makefile

```
@@ -29,7 +30,22 @@ pylint:
 flake8:
 	flake8 .
 
-test: lint unit-test
+test: lint unit-test tear-down-integration integration-test
 
 unit-test:
-	pytest
+	pytest --ignore=assisted-events-scrape/tests/integration
+
+integration-test: build-image
+	kind create cluster --name assisted-events-scrape || true
+	kind --name assisted-events-scrape export kubeconfig
+	kind load docker-image --name assisted-events-scrape $(ASSISTED_EVENTS_SCRAPE_IMAGE)
+	kubectl apply -f dev-manifests
+	./tools/generate_mockserver_configmap | kubectl delete -f - || true
+	./tools/generate_mockserver_configmap | kubectl create -f - || true
+	./tools/ocp2k8s $(ASSISTED_EVENTS_SCRAPE_IMAGE) | kubectl apply -f -
+	kubectl wait --timeout=120s --for=condition=Available deployment --all
+	kubectl wait --timeout=300s --for=condition=Ready pods --all
+	ES_SERVER=$$(./tools/get_endpoint) ES_INDEX=assisted-service-events pytest --ignore=assisted-events-scrape/tests/unit
```

Same as above, better be inclusive than exclusive:
```suggestion
	ES_SERVER=$$(./tools/get_endpoint) ES_INDEX=assisted-service-events pytest assisted-events-scrape/tests/integration/
```